Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine
+
+RUN apk add py-pip
+RUN pip install aws-exporter
```

Comment:
should install that from local sources I guess

Reply:
that means you'd need to make sure the .git dir is copied to the container as well and removed after install

Reply:
that's an option but why not just use a pip?